content/applications/finance/payment_acquirers.rst

```diff
@@ -168,6 +168,11 @@ authenticate the payment, or if to stay on the current page and authenticate the
 
 .. note::
    Some features are available only if you select *Redirection to the acquirer website*.
+.. warning::
+   When configuring a payment acquirer with the payment flow **Payment from Odoo**, the payment
+   acquirer will not be visible to people who do not have an account on the Odoo portal. To add a
```

Comment:
When configuring a payment acquirer with the payment flow **Payment from Odoo, such acquirer will not be visible to people who do not have an account on the Odoo portal.